lib/modules/platform/github/index.ts

```diff
@@ -1731,6 +1732,25 @@ export async function updatePr({
   }
 }
 
+export async function reattemptPlatformAutomerge({
+  prNo,
+  platformOptions,
+}: ReattemptPlatformAutomergeConfig): Promise<void> {
+  try {
+    const result = await getPr(prNo);
```

Comment:
```suggestion
    const result = (await getPr(prNo))!;
```

I think we can assume that the pr exists